src/main/java/org/mitre/synthea/export/CSVExporter.java

```diff
@@ -480,7 +510,9 @@ public void export(Person person, long time) throws IOException {
 
       for (Medication medication : encounter.medications) {
         medication(personID, encounterID, payerID, medication, time);
-        claim(person, medication.claim, encounter, encounterID, time);
+        if (Config.getAsBoolean("exporter.csv.claim_transactions")) {
+          claim(person, medication.claim, encounter, encounterID, time);
+        }
```

Comment:
Not sure about these `if` code blocks and the two new parameters... if you don't want those files, then intent was to for you to use the `exporter.csv.excluded_files` property. For example,

```properties
# included_files and excluded_files list out the files to include/exclude in the csv exporter
# only one of these may be set at a time, if both are set then both will be ignored
# if neither is set, then all files will be included
# see list of files at: https://github.com/synthetichealth/synthea/wiki/CSV-File-Data-Dictionary
# include filenames separated with a comma, ex: patients.csv,procedures.csv,medications.csv
# NOTE: the csv exporter does not actively delete files, so if Run 1 you included a file, then Run 2 you exclude that file, the version from Run 1 will still be present
exporter.csv.included_files =
exporter.csv.excluded_files = claims.csv,claims_transactions.csv,patient_expenses.csv
```


I suppose using `exporter.csv.claim_transactions` and `exporter.csv.patient_expenses` is marginally faster, but it is special casing the claims processing.